docs/spec.openapi.yml

```diff
@@ -0,0 +1,658 @@
+openapi: 3.0.3
+
+info:
+  title: Capyfile Gateway Service
+
+  license:
+    name: MIT
+    url: https://github.com/hawks-atlanta/gateway-java/blob/main/LICENSE
+
+  version: TBA
+  
+  description: |
+    Describes requests exposed in the gateway SOAP service. Despite it not being a REST API this specification was created for developers convenience, so that they don't have to deal with the WSDL file. This document is for reference only.
+    
+    How to read this document:
+    - Endpoints → SOAP requests
+    - Response codes → ResStatus code field
+
+tags:
+  - name: Auth
+  - name: Account
+  - name: File
+  - name: Share
+
+paths:
+
+  # authorization / authentication
+  
+  /auth_login:
+    post:
+      tags:
+        - Auth
+      description: Authenticates to the server
+      requestBody:
+        content:
+          Object:
+            schema:
+              $ref: '#/components/schemas/Credentials'
+      responses:
+        '200':
+          description: Login succeeded
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResSession'
+        'default':
+          description: An error occurred, see other codes at [Authentication API Spec](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+
+                
+  /auth_refresh:
```

Comment:
I think there is no `/refresh` endpoint in the auth micro-service

Reply:
The `/challenge` endpoint effectively acts as a `/refresh`. It returns a new token.